### PR TITLE
Celo integration

### PR DESCRIPTION
**Pull request by enjeyw**
**Describe the Pull Request**

Adding compatibility with Celo.
Celo eth_account fork can be installed via pip install git+https://github.com/teamsempo/eth-account.git@celo

**Todo**

- [ ] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [ ] Request review from relevant @person or team
- [ ] QA your pull request. This includes running the app, login, testing changes etc.
- [ ] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: config_files/public/local_config.ini [hunk 1]
```diff
@@ -66,6 +66,31 @@ reserve_token_address         =
 reserve_token_name            = Kenyan Shilling
 reserve_token_symbol          = Ksh
 
+[CELO]
+;contract_type is one of "standard, ccv, mintable. Defaults to standard"
+contract_type           = standard
+contract_address        = 0x874069fa1eb16d44d622f2e0ca25eea172369bc1
+http_provider           = https://alfajores-forno.celo-testnet.org
+websocket_provider      = ws://alfajores-forno.celo-testnet.org:8546
```
Comment (michielderoos) [inline config_files/public/local_config.ini:74]: Is this websocket address tested for tx sync?

File: devtools/quick_setup_script.sh [hunk 2]
```diff
@@ -129,11 +129,11 @@ echo ~~~Setting up Contracts
...
 
-if [[ "mockData" != 'none' ]]; then
-    echo ~~~Creating mock data
-    cd ./app/server/utils
-    python -u mock_data.py ${testdata}
-fi
+#if [[ "mockData" != 'none' ]]; then
+#    echo ~~~Creating mock data
+#    cd ./app/server/utils
+#    python -u mock_data.py ${testdata}
```
Comment (michielderoos) [inline devtools/quick_setup_script.sh:135]: Are these commented out for testing, or are they broken with this PR?

**Review by michielderoos — CHANGES_REQUESTED** [review]
Have a few comments/notes, but it looks clean!

File: eth_worker/eth_src/eth_manager/eth_transaction_processor.py [hunk 3]
```diff
@@ -129,17 +129,26 @@ def _process_transaction(self,
             private_key=signing_wallet_obj.private_key
         )
 
+        # Save the hash and nonce before attempting txn
```
[the remark this answers is not in the document]
Comment (enjeyw) [reply eth_worker/eth_src/eth_manager/eth_transaction_processor.py:132]: oh no this is actually desired bheaviour beaccaaaaauseeeeee there's a decent chance of failure on 
```
        self._send_signed_transaction(signed_transaction, transaction_id)
```
so we want to track if it fails before or after!

File: config_files/public/local_config.ini (hunk 1 above)
Comment (enjeyw) [reply config_files/public/local_config.ini:74]: nope lol! was gonna do on the dev instance.

File: devtools/quick_setup_script.sh (hunk 2 above)
Comment (enjeyw) [reply devtools/quick_setup_script.sh:135]: oh they're just broken in general, but this was me forgetting to uncomment

File: eth_worker/eth_src/eth_manager/eth_transaction_processor.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (enjeyw) [reply eth_worker/eth_src/eth_manager/eth_transaction_processor.py:132]: maybe we need an extra lifecycle option

[the remark this answers is not in the document]
Comment (enjeyw) [reply eth_worker/eth_src/eth_manager/eth_transaction_processor.py:132]: UNSTARTED -> CONSTRUCTED -> SUBMITTED -> PENDING -> COMPLETE 
or something to that effct

**Review by michielderoos — APPROVED** [review]
LGTM! (Merge pending config files being tweaked, and QA on celo.withsempo)

**Pull request merged**